gate-oauth2/src/test/java/com/netflix/spinnaker/gate/security/oauth2/OAuthUserInfoServiceHelperTest.java

```diff
@@ -48,9 +46,22 @@ public class OAuthUserInfoServiceHelperTest {
 
   @Mock private Front50Service front50Service;
 
+  private OAuthUserInfoServiceHelper userInfoService;
+
   @BeforeEach
   void setUp() {
     MockitoAnnotations.openMocks(this);
```

Comment:
Not technically in scope for this PR, but don't we need a new member like:
```
private AutoCloseable mocks;
```
and then here:
```
mocks = MockitoAnnotations.openMocks(this);
```
with a new
```
@AfterEach
void cleanup() {
  mocks.close();
}
```
or likely simpler than all this is annotating the class with
```
@ExtendWith(MockitoExtension.class)
```
and removing
```
MockitoAnnotations.openMocks(this);
```

Reply:
Thank you @dbyron-sf 

Here is the update https://github.com/spinnaker/gate/pull/1887/commits/c08c328bb87568b60b0d2d3b3cd5878d56a1a6a2